inference-engine/src/transformations/include/transformations/depth_to_space_fusion.hpp

```diff
@@ -12,23 +12,28 @@
 #include <ngraph/pass/graph_rewrite.hpp>
 #include "transformations/utils/pass_param.hpp"
 
+/**
+ * @brief ngraph namespace
+ */
 namespace ngraph {
+/**
+ * @brief ngraph::pass namespace
+ */
 namespace pass {
```

Comment:
Locally on my machine I see a lot of warnings from doxygen that some methods, etc are not documented like:
```
C:/Users/ilavreno/Documents/Programming/git_repo/dldt/inference-engine/src/transformations/include/transformations/utils/pass_param.hpp:21: warning: Member param_callback (typedef) of class ngraph::pass::PassParam is not documented.
C:/Users/ilavreno/Documents/Programming/git_repo/dldt/inference-engine/src/transformations/include/transformations/utils/pass_param.hpp:23: warning: Member PassParam(const param_callback &callback=getDefaultCallback()) (function) of class ngraph::pass::PassParam is not documented.
C:/Users/ilavreno/Documents/Programming/git_repo/dldt/inference-engine/src/transformations/include/transformations/utils/pass_param.hpp:25: warning: Member setCallback(const param_callback &callback) (function) of class ngraph::pass::PassParam is not documented.
C:/Users/ilavreno/Documents/Programming/git_repo/dldt/inference-engine/src/transformations/include/transformations/utils/pass_param.hpp:29: warning: Member getDefaultCallback() (function) of class ngraph::pass::PassParam is not documented.
C:/Users/ilavreno/Documents/Programming/git_repo/dldt/inference-engine/src/transformations/include/transformations/utils/pass_param.hpp:36: warning: Member transformation_callback (variable) of class ngraph::pass::PassParam is not documented.
...
```
Can we fix all of them?

Reply:
Some of them were fixed and others will be fixed in other PR.